src/Bundle/CoverallsBundle/Repository/JobsRepository.php

```diff
@@ -157,14 +163,15 @@ protected function dumpJsonFile()
 
     /**
      * Send json_file to Jobs API.
+     *
+     * @return bool
      */
     protected function send()
     {
         $this->logger->info(sprintf('Submitting to %s', Jobs::URL));
 
         try {
             $response = $this->api->send();
```

Comment:
why, why `Api::send` is checking dry-run and returns `null` in such case :( it shall be not called if it's dry-run...